transcendence-app/src/user/dto/user.dto.ts

```diff
@@ -1,6 +1,8 @@
 import { IsEmail, IsNotEmpty, IsString, IsUrl } from 'class-validator';
 
 export class UserDto {
+  id: number | undefined;
+
```

Comment:
I don't have any particular preference on which of these two do we like.
If we had users other than from 42, I'd say it makes more sense to use different Ids, otherwise, intranet Id works just as well.
I'll be trying the second one.

Reply:
But what about we do this in a following PR?

Reply:
Sure, as you see fit.

Reply:
I really align with the idea of getting the userId from the db, we have to control our data and a `userId` is a very sensitive one. If we use the one coming from 42API we'll be limiting ourselves in many ways:

what if we decide to implement another type of logging? what if userId intra42 specification changes?

Due to this, I'd genereate our own ids and, remove the `userId` from the dto. If we follow this approach it'll be useless

Reply:
It's temporary. I intend to do this
```
Or create a new class UserEntity/UserModel, which will map to the User table inside the database and add all the additional fields (like id) to that class.
```
in a follow up PR.

Reply:
The follow up PR is [this one](https://docs.nestjs.com/techniques/database).